x-pack/test_serverless/api_integration/test_suites/observability/cases/get_case.ts

```diff
@@ -47,7 +47,22 @@ export default ({ getService }: FtrProviderContext): void => {
 
       expect(data).to.eql(expectedData);
       expect(createdBy).to.have.keys('full_name', 'email', 'username');
-      expect(data.comments?.length).to.eql(0);
+    });
+
+    it('should throw a 400 if the query param includeComments is being used', async () => {
```

Comment:
nit: Could you also add this to `x-pack/test_serverless/api_integration/test_suites/security/cases/get_case.ts`?

Reply:
Damn I forgot about it. Done in [`9319e4e` (#198378)](https://github.com/elastic/kibana/pull/198378/commits/9319e4e58f6dc272113b3dd86b446e626d3e7839)